### PR TITLE
refactor: integrate "Add to MetaMask" buttons for zkSync networks 

### PR DESCRIPTION
**Pull request by daweilv**
# What :computer: 
* This PR forked from  https://github.com/matter-labs/zksync-web-era-docs/pull/935
* The previous PR added "Add to MetaMask" buttons for zkSync Era Mainnet and Sepolia testnet network configurations
* This PR convert the vanilla javascript&html code into a Vue3 component

# Evidence :camera:
image
image

# Notes :memo:
* na

### CODE REVIEW

File: docs/.vuepress/components/NetworkAdder.vue [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+}
+</script>
+
+<style scoped>
+.add-network {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: 4px 8px;
+  gap: 4px;
```
Comment (uF4No) [inline docs/.vuepress/components/NetworkAdder.vue:49]:
```suggestion
  gap: 4px;
  cursor: pointer;
```

File: docs/build/quick-start/add-zksync-to-metamask.md [hunk 2]
```diff
@@ -18,6 +18,8 @@ Fill in the following details for the zkSync network:
 
 ## Mainnet network info
 
+<network-adder network="mainnet"></network-adder>
+
```
Comment (uF4No) [inline docs/build/quick-start/add-zksync-to-metamask.md:22]:
```suggestion
Add zkSync Era Mainnet automatically using the button below:

<network-adder network="mainnet"></network-adder>

**Network details:**

```

**Comment by uF4No** [comment]
thanks for the contribution @daweilv . I've added a few minor comments. Please review and update 🤙

**Comment by daweilv** [comment]
> thanks for the contribution @daweilv . I've added a few minor comments. Please review and update 🤙

Updated. Much better UX! @uF4No

**Review by uF4No — APPROVED** [review]

**Pull request merged**